app/src/main/res/xml/preferences_playback.xml

```diff
@@ -71,6 +71,11 @@
                 android:key="prefPlaybackSpeedLauncher"
                 android:summary="@string/pref_playback_speed_sum"
                 android:title="@string/pref_playback_speed_title"/>
+        <SwitchPreference
+            android:defaultValue="false"
+            android:key="prefPlaybackDurationRespectSpeed"
+            android:summary="@string/pref_playback_time_respect_speed_sum"
+            android:title="@string/pref_playback_time_respect_speed_title"/>
```

Comment:
Please use the same indentation that is already used in the file

Reply:
oups, sorry about that